### PR TITLE
feat: Migrate 'About us' static content

### PR DESCRIPTION
**Pull request by abbyoung**
# SC-108

## Proposed changes

* Move About Us page to new beta layout

## Reviewer notes
* Need guidance on placement and styling of the 'About this portal' link that should be moved to the footer.
* Does it need to go in this PR?
  * Yes -- added link to the footer as first item in list

## Setup

```sh
    yarn services:up
    yarn dev
 ```

Visit `localhost:3000/about-us` or view on [dev](https://dev.ussforbit.us/about-us) at `/about-us` once you've joined the beta.

### As the original developer, I have

- [x] Met the acceptance criteria, or will meet them in subsequent PRs or stories
  - ... 
- ~[ ] Created new stories in Storybook if applicable~
  - ~[ ] Checked that all Storybook accessibility checks are passing~
- ~[ ] Created/modified automated unit tests in Jest~
  - ~[ ] Including jest-axe checks when UI changes~
- [x] Created/modified automated E2E tests in Cypress
  - [x] Including cypress-axe checks when UI changes
- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):
  - [x] Checked responsiveness in mobile, tablet, and desktop
  - [x] Checked keyboard navigability
  - [x] Tested with [VoiceOver](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) in Safari
  - [x] Checked VO's [rotor menu](https://github.com/trussworks/accessibility/blob/master/README.md#how-to-use-the-rotor-menu) for landmarks, page heading structure and links
  - [x] Used a browser a11y tool to check for issues (WAVE, axe, ANDI or Accessibility addon tab for Storybook)
- [x] Requested a design review for user-facing changes
- For any new migrations/schema changes:
  - ~[ ] Followed guidelines for zero-downtime deploys~

### As code reviewer(s), I have

- [x] Pulled this branch locally and tested it
- [x] Reviewed this code and left comments
- [x] Checked that all code is adequately covered by tests
- [x] Made it clear which comments need to be addressed before this work is merged
- [x] Considered marking this as accepted even if there are small changes needed
- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):
  - [x] Checked responsiveness in mobile, tablet, and desktop
  - [x] Checked keyboard navigability
  - [x] Tested with [VoiceOver](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) in Safari
  - [x] Checked VO's [rotor menu](https://github.com/trussworks/accessibility/blob/master/README.md#how-to-use-the-rotor-menu) for landmarks, page heading structure and links
  - [x] Used a browser a11y tool to check for issues (WAVE, axe, ANDI or Accessibility addon tab for Storybook)

### As a designer reviewer, I have

- [ ] Checked in the design translated visually
- [ ] Checked behavior
- [ ] Checked different states (empty, one, some, error)
- [ ] Checked for landmarks, page heading structure, and links
- [ ] Tried to break the intended flow
- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):
  - [ ] Checked responsiveness in mobile, tablet, and desktop
  - [ ] Checked keyboard navigability
  - [ ] Tested with [VoiceOver](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) in Safari
  - [ ] Checked VO's [rotor menu](https://github.com/trussworks/accessibility/blob/master/README.md#how-to-use-the-rotor-menu) for landmarks, page heading structure and links
  - [ ] Used a browser a11y tool to check for issues (WAVE, axe, ANDI or Accessibility addon tab for Storybook)

---

### CODE REVIEW

**Comment by jbecker01** [comment]
@abbyoung no need to include launches in this release

**Comment by jbecker01** [comment]
@abbyoung all the pieces are there and this looks good to me. I'll wait to hear back from design on the visual component. 

Also, design will be looking into "styling of the 'About this portal' link that should be moved to the footer." 
I'm not sure it needs to go into this PR. @suzubara should we keep these two items in the same PR or open a new issue to cover this?

**Comment by jbecker01** [comment]
@abbyoung I had a quick chat with Suz this morning and decided its best to add the About this site URL to the footer in this PR and then we'll open up a new ticket to handle styling.

File: e2e/cypress/integration/about-us.spec.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import logging from '../plugins/logging'
+
+describe('About Us', () => {
+  before(() => {
+    // Reset the database
+    cy.task('db:seed')
```
Comment (suzubara) [inline e2e/cypress/integration/about-us.spec.js:6]: I don't think we need to reset the db for this test

**Review by suzubara — APPROVED** [review]

**Pull request merged**